integration-tests/load/ocr/config.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/pelletier/go-toml/v2"
 	"github.com/rs/zerolog/log"
 
-	"github.com/smartcontractkit/chainlink/v2/core/store/models"
+	commonconfig "github.com/smartcontractkit/chainlink-common/pkg/config"
```

Comment:
Can we avoid aliasing when there is no conflict?
```suggestion
	"github.com/smartcontractkit/chainlink-common/pkg/config"
```